client/src/app/shared/shared-main/video-channel/video-channel.model.ts

```diff
@@ -74,6 +75,10 @@ export class VideoChannel extends Actor implements ServerVideoChannel {
       this.viewsPerDay = hash.viewsPerDay.map(v => ({ ...v, date: new Date(v.date) }))
     }
 
+    if (hash.totalViews) {
+      this.totalViews = hash.totalViews
```

Comment:
`if (hash.totalViews !== null && hash.totalViews !== undefined)` since it could be `0`